src/api/pros.py

```diff
@@ -0,0 +1,99 @@
+from flask import Blueprint, jsonify, request, abort
+import uuid
+import datetime
+import json
+from utils import load_json, save_json
+
+pros = Blueprint('pros', __name__, url_prefix='/pros')
+
+
+@pros.route('/', methods=['GET'])
+def list_pros():
+    """
+    Fetches list of professionals
+    """
+    db = load_json('./src/api/db.json')
```

Comment:
Generally, any hardcoded value like like path or global constants should be defined in a separate _config.py_. Check out DRY paradigm. It's a good practice.

This need not be changed for now, because we'll be rewriting this part anyway. Just wanted to share for your learning.

Reply:
I'll take care of this later, thanks for sharing though.